<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-KKPJ5RQ" failures="0" tests="4" name="com.Test.KIIJKSluiter_WelcomePage" time="0.084" errors="1" timestamp="2023-09-10T15:43:19 IST" skipped="3">
  <testcase classname="com.Test.KIIJKSluiter_WelcomePage" name="BS" time="0.084">
    <error message="C:\KIJKSluiterProject\KIJKSluiterProj\TestData\ExcelData.xlsx (The system cannot find the path specified)" type="java.io.FileNotFoundException">
      <![CDATA[java.io.FileNotFoundException: C:\KIJKSluiterProject\KIJKSluiterProj\TestData\ExcelData.xlsx (The system cannot find the path specified)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
at com.Utility.ExcelDataProvider.<init>(ExcelDataProvider.java:14)
at com.Utility.BaseClass.BS(BaseClass.java:30)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:390)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:325)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:382)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- BS -->
  <testcase classname="com.Test.KIIJKSluiter_WelcomePage" name="verifyLoginPage" time="0.000">
    <skipped/>
  </testcase> <!-- verifyLoginPage -->
  <testcase classname="com.Test.KIIJKSluiter_WelcomePage" name="verifyKIJKSluiterPage" time="0.000">
    <skipped/>
  </testcase> <!-- verifyKIJKSluiterPage -->
  <testcase classname="com.Test.KIIJKSluiter_WelcomePage" name="verifyMedicinePage">
    <skipped/>
  </testcase> <!-- verifyMedicinePage -->
</testsuite> <!-- com.Test.KIIJKSluiter_WelcomePage -->
